presto-product-tests/bin/run_on_docker.sh

```diff
@@ -134,9 +125,7 @@ trap terminate INT TERM EXIT
 # display how test environment is configured
 environment_compose config
 
-environment_compose up -d
-environment_compose logs --no-color -f &
```

Comment:
Are these equivalent? It seems that the `logs -f` command could continue to print logs after the container has started. If so, there might be valuable messages, such as an OOM from one of the Hadoop processes.

Reply:
They are roughly equivalent.
Old way (`up -d` + `logs -f`) could (theoretically?) miss some early log messages.
New way (`up &`) shows all the logs.